octobot_trading/personal_data/portfolios/portfolio_util.py

```diff
@@ -100,15 +100,15 @@ async def get_coefficient_of_determination_data(transactions, start_balance,
             if hasattr(transaction, "quantity"):
                 current_pnl = float(transaction.quantity)
                 pnl_history_times.append(transaction.creation_time)
-            elif hasattr(transaction, 'realised_pnl'):
-                current_pnl = float(transaction.realised_pnl)
+            elif hasattr(transaction, "realized_pnl"):
+                current_pnl = float(transaction.realized_pnl)
                 pnl_history_times.append(transaction.creation_time)
             elif isinstance(transaction, dict):
                 if transaction["quantity"]:
                     current_pnl = transaction["quantity"]
                     pnl_history_times.append(transaction["x"])
-                elif transaction['realised_pnl']:
-                    current_pnl = transaction['realised_pnl']
+                elif transaction["realized_pnl"]:
+                    current_pnl = transaction["realized_pnl"]
                     pnl_history_times.append(transaction["x"])
```

Comment:
Why do we have to check transaction attributes?

Reply:
I didnt change any logic here, just fixed the typo realiSed / realiZed to have it consistent all over octobot

Reply:
Yes, but I was asking about the logic

Reply:
I dont know either, havent looked into transactions yet

Reply:
I dont know either, havent looked into transactions yet.